### DISCUSSION TITLE
Need Multi-App advice

### DISCUSSION THREAD

**Discussion by jameshhood** [ionic-team/capacitor#6496] [Q&A]
I have a VueJS app that we host for several different customers which is all configured through different CMS servers. This yields one build which then the server running it can then pass environment variables to it as it's ran. Fast forward to adding capacitor to the project.

I need to be able to have a capacitor app for each customer of ours yet by default you can only have a single ios/android app per project. What's the best way to handle this? Currently I have a CI/CD pipeline that will remove the ios directory, copy over the customers capacitor.config.js, then `yarn add @capacitor/ios` to regenerate the ios folder. This doesn't allow me to configure anything via xcode. Currently I'm having to figure out how to enable fullscreen for iPads because I can't push the build generated this way to the app-store.

It's already a pain to get capacitor to work with MS AppCenter, but now I'm getting to the point where I'm pretty much forced to have a monorepo and I really don't want one if I can help it

**Comment by mlynch** [comment]
Take a look at trapeze: https://trapeze.dev/

It can automate a lot of these kinds of white labeled app tasks, and works great with our Appflow CI/CD service (here's an example on how to run those workflows in CI/CD: https://trapeze.dev/docs/ci-cd). AFAIK MS app center doesn't officially support Capacitor

**Reply by jameshhood** [reply]
> AFAIK MS app center doesn't officially support Capacitor

Yeah it doesn't because each project would have to be in their own repo to trigger a build. However, you can still use command line to build the ipa/aab's required to publish a build and push those directly to AppCenter.

This is all kind of new territory for me so I'm not even sure that's worth it. The idea there was to be able to push development builds every time source changed and then we can promote from there to app-stores for actual testing.

> Take a look at trapeze: https://trapeze.dev/

So I guess I would just make my respective ios/android projects and configure them as such and then this could configure the version, appIDs, ect for me? I would just need to use `cordova-res` to generate the respective icons/splash screens
